fix(docs): singleton item update docs mention primary key (#24430)
directus/directus: docs/reference/items.md

### Response

> Returns an [item object](#the-item-object) if a valid collection name was provided.

Reviewer: ```suggestion
Returns the [item object](#the-item-object) of the singleton that was updated.
```
